molecule/driver/delegated.py

```diff
@@ -89,8 +89,8 @@ class Delegated(base.Base):
             managed: False
             login_cmd_template: 'ssh {instance} -F /tmp/ssh-config'
             ansible_connection_options:
-              ansible_connection: ssh
-              ansible_ssh_common_args -F /path/to/ssh-config
+                ansible_connection: ssh
```

Comment:
We don't need to have extra indentation here. Two spaces is pretty typical for yaml